dockers/docker-orchagent/ports.json.j2

```diff
@@ -1,13 +1,14 @@
 [
 {% if PORT %}
+{% set ns = {'firstPrinted': False} %}
 {% for port in PORT %}
 {% if PORT[port].has_key('speed') %}
-    {
+    {% if ns.firstPrinted %},{% endif %}{
         "PORT_TABLE:{{ port }}": {
             "speed": "{{ PORT[port]['speed'] }}"
         },
         "OP": "SET"
-    }{% if not loop.last %},{% endif %}
+    }{% if ns.update({'firstPrinted': True}) %} {% endif %}
```

Comment:
do we need another if here, we can probably do { % set ns = {'firstPrinted': True} %}

Reply:
Unfortunately, we can't change variable in a loop in a straight way. You couldn't be able to notice that changes inside the loop itself.
You also can not simply call the function like {% ns.update({'firstPrinted': True}) %}. It leads to error and ports.json is empty as a result.
The main idea was that this function must be called to calculate that if condition. Unfortunately, I have not find another way to do it.